src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -77,11 +74,9 @@
 (defn- emoji-row
   [row-data {:keys [on-select close]}]
   (into [rn/view {:style style/emoji-row-container}]
-        (map-indexed
-         (fn [col-index {:keys [hexcode] :as emoji}]
-           ^{:key hexcode}
-           [emoji-item emoji col-index on-select close])
-         row-data)))
+        (map-indexed (fn [col-index emoji]
+                       [emoji-item emoji col-index on-select close]))
+        row-data))
```

Comment:
Another improvement in performance in two aspects:

#### 1. We don't need `hexcode` as key, 

we need setting a `:key` when we return a sequence of elements, but this function is using `into`, so we are just returning hiccup.

  Just to explain it better, we need `:key` metadata when:
```clojure
[rn/something
  ( ;; <- a sequence
    ^{:key 1} [rn/something-else ...]
    ^{:key 2} [rn/something-else ...]
    ^{:key 3} [rn/something-else ...])]
```

since we are using `into`, we are returning just valid hiccup, no need of `:key`
```clojure
[rn/something
 [rn/something-else ...]
 [rn/something-else ...]
 [rn/something-else ...])]
```
#### 2. Using xform parameter of `into`

`into` allows the use of transducers (a transformation to data in a more performant way), and this piece of code was perfect for it, since we have:
1. something we want to move into (`[rn/view...]`)
2. a transformation of data (`(map-indexed (fn [...] ...))`) that also returns a transducer when called with no params
3. data (`row-data`).

So, I'm using `into` as:
`(into [] [transformation data)`
which is:
`(into [rn/view...] (map-indexed ...) row-data)`


If you'd like to read more about transducers:
https://clojure.org/reference/transducers
> into
> To apply a transducer to an input collection and construct a new output collection, use [into](https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/into) (which efficiently uses reduce and transients if possible):

Reply:
Thanks for the detailed explanation! @ulisesmac.

We need to keep the `key` in this case, as it's used to track views (when the items change) when the user is searching. If we remove the `key`, the list will see the last rendered item. 

https://github.com/status-im/status-mobile/assets/19339952/dc4caa36-1a87-4a27-aa9b-9bf12433fb37

We can't use `col-index` as it will be the same for every row. But, the `hexcode` is unique.

Reply:
@smohamedjavid  Thanks I fixed it!

IMO using hexocde is a nice approach!